src/Assert.php

```diff
@@ -1566,8 +1566,8 @@ public static function interfaceExists($value, $message = '')
     /**
      * @psalm-pure
      * @psalm-template ExpectedType of object
-     * @psalm-param class-string<ExpectedType> $interface
-     * @psalm-assert class-string<ExpectedType> $value
+     * @psalm-param class-string<ExpectedType>|ExpectedType $interface
```

Comment:
The `$interface` cannot be an object, because `in_array` will fail. Only the `$value` can be an object.

Reply:
It has been a while 😅 good find! I adapted it, not sure if that is/was enough here, let's see